src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -39,6 +43,9 @@ public BlobFeedAction(string expectedFeedUrl, string accountKey, MSBuild.TaskLog
                 string relativePath = m.Groups["relativepath"].Value;
                 feed = new BlobFeed(accountName, accountKey, containerName, relativePath, Log);
                 feedUrl = m.Groups["feedurl"].Value;
+                retries = retryAttempts;
+                delay = TimeSpan.FromSeconds(retryDelay);
```

Comment:
Do we think we need some random variation in the retry delay?

Reply:
I've added logic below to increase the delay on every iteration since, depending on what's being pushed it might take up to minutes. I don't think randomizing will make a difference but making the delay longer

Reply:
In other BT code we've done exponential stepback, but not randomness (yet):

https://github.com/dotnet/buildtools/blob/fffbc00cd3e7cc8bd5724a662e4c1a4c2cf8755f/src/Microsoft.DotNet.Build.Tasks/ExecWithRetries.cs#L26

https://github.com/dotnet/buildtools/blob/fffbc00cd3e7cc8bd5724a662e4c1a4c2cf8755f/src/Microsoft.DotNet.Build.Tasks/PackageFiles/scripts/docker/init-docker.sh#L22

I don't think anyone's actually experimented to get there, but it kinda makes sense that the first tries should be relatively quick, then later make them longer to account for potential network congestion.